### ISSUE TITLE
Jackett releases question

### ISSUE THREAD

**Issue by ngosang** [Jackett/Jackett#7183]
Why we have 2 bots generating drafs of releases?
*  jackett-bot => v0.13.XX
*  jackettbot => v0.12.XX

@garfield69 @junglebus @cadatoiva

**Comment by garfield69** [comment]
junglebus did not have access to the jackettbot user so he created the jackett-bot user for the azure pipe publishing.
I presume that once we drop the appveyor builds the jackettbot will be discontinued.

**Comment by cadatoiva** [comment]
This was my understanding as well

**Comment by junglebus** [comment]
I've tested the auto updater for Windows. Tested updating from the Appveyor build (full framework) to the Azure Pipelines build (.NET Core Windows) and it worked perfectly

I think we are ready to start making 0.13.x as pre-releases now. Unlikely anything will break with the .NET Core (Linux and macOS) builds. Windows is a pretty significant change going from full framework to Core, but its been working well for me. Mono, smaller risk of breaking, but there is change now that we are building the Mono version on Linux now instead of Windows
